test/integration/main_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2023 The Authors (see AUTHORS file)
```

Comment:
There is already a [test/integ](https://github.com/abcxyz/jvs/tree/main/test/integ) folder. We might want to rethink how we are structuring these.

Reply:
I am thinking we only need the test/integration, the test/integ is removed. What is your suggestion?

Reply:
Hey @sqin2019 - sorry, I missed this comment in my notifications 😦 

Taking a step back: what exactly are we trying to integration test? If https://github.com/abcxyz/jvs/pull/245 is merged, there's a shared framework for starting all the services.

Should we move the integration tests closers to the servers? The tests in #245 already do a basic health check, and they could be expanded to do more if needed.

Reply:
Hi @sethvargo no worries! 

For the api path, we can use jvsctl to call each API and verify the output, will use the new framework for this.

For moving the integration tests closer to the servers, do you mean to split them up and move them to the corresponding server packages? 

For expanding the health check, do you mean to expand and add it in the integration tests? If so, it can be a follow up once integration tests is ready.

Reply:
> For moving the integration tests closer to the servers, do you mean to split them up and move them to the corresponding server packages?

Yes

> For expanding the health check, do you mean to expand and add it in the integration tests? If so, it can be a follow up once integration tests is ready.

Let's table this for a longer discussion.